common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -841,4 +841,5 @@ describe 'Problem', ->
       jasmine.clock().tick(64000)
       expect(@problem.poll.calls.count()).toEqual(6)
 
-      expect($('.capa_alert').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
+      expect($('.notification-gentle-alert .notification-message').is(':visible')).toBe(true)
```

Comment:
Yes, I also tested that this will work by temporarily removing the .show method on the notification.